openreview/openreview.py

```diff
@@ -222,29 +222,33 @@ def send_mail(self, subject, recipients, message):
 
         return response
 
-    def add_members(self, groupId, members):
-        def add_member(group,members):     
-            response = requests.put(self.groups_url+'/members', json={'id':group,'members':members}, headers=self.headers)
-            response = self.__handle_response(response)
-            return self.get_group(response.json()['id'])
-
-        if type(members)==str:
-            return add_member(groupId,[members])
-        if type(members)==list:
-            return add_member(groupId,members)
+    def add_members_to_group(self, group, members):
+        if self.exists(group.id):
```

Comment:
it is not necessary the validation, because the API return an error if the groups does not exist.